src/HH4b/postprocessing/PostProcess.py

```diff
@@ -443,6 +462,25 @@ def load_process_run3_samples(args, year, bdt_training_keys, control_plots, plot
                 weight_ttbar=args.weight_ttbar_bdt,
                 bdt_disc=args.bdt_disc,
             )
+
+            # redefine VBF variables
```

Comment:
so we're just doing this for plotting purposes, but not the actual inference, right? just checking